meson.build

```diff
@@ -1195,6 +1195,18 @@ else
 endif
 conf.set10('HAVE_LZ4', have)
 
+want_zstd = get_option('zstd')
+if want_zstd != 'false'
```

Comment:
Could you add "and not skip_deps" here? Almost all the dependencies are skipped automatically when the fuzz targets are built.

Reply:
Dont know what I am doing, but sure